.devcontainer/post-start.sh

```
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+
+echo "🔄 Starting development environment..."
+
+# Ensure all services are running
+echo "📡 Checking service health..."
+
+# Check PostgreSQL
+until PGPASSWORD=$POSTGRES_PASSWORD psql -h db -U $POSTGRES_USER -d $POSTGRES_DB -c '\q' 2>/dev/null; do
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)